include/vapor/UnstructuredGrid.h

```diff
@@ -127,23 +127,24 @@ class VDF_API UnstructuredGrid : public Grid {
     static std::string GetClassType() { return ("Unstructured"); }
     std::string        GetType() const override { return (GetClassType()); }
 
-    bool GetCellNodes(const Size_tArr3 &cindices, std::vector<Size_tArr3> &nodes) const override;
+    bool GetCellNodes(const DimsType &cindices, std::vector<DimsType> &nodes) const override;
 
     //! \copydoc Grid::GetCellNeighbors()
     //!
-    virtual bool GetCellNeighbors(const Size_tArr3 &cindices, std::vector<Size_tArr3> &cells) const override;
+    virtual bool GetCellNeighbors(const DimsType &cindices, std::vector<DimsType> &cells) const override;
 
     //! \copydoc Grid::GetNodeCells()
     //!
-    virtual bool GetNodeCells(const Size_tArr3 &indices, std::vector<Size_tArr3> &cells) const override;
+    virtual bool GetNodeCells(const DimsType &indices, std::vector<DimsType> &cells) const override;
 
     size_t GetMaxVertexPerFace() const override { return (_maxVertexPerFace); };
 
     size_t GetMaxVertexPerCell() const override { return ((GetTopologyDim() == 3) ? 2 * GetMaxVertexPerFace() : GetMaxVertexPerFace()); };
 
     //! Return the grid node dimmensions
     //!
-    const std::vector<size_t> &GetNodeDimensions() const override { return (_vertexDims); }
+    const std::array<size_t, 3> GetNodeDimensions() const override;
```

Comment:
Shouldn't this be DimsType?

Reply:
You're right. Fixed this one together with a few others that were missed.